### PR TITLE
Update the dashboard feed to use new API endpoint from Data Hub help centre

### PR DESCRIPTION
**Pull request by ian-leggett**
## Description of change
We need to stop using the Zen desk API feed for the homepage dashboard and use the new feed from the Data Hub Help centre. 

## Test instructions

You should be able to see the new feed pulling through to the homepage dashboard, it should reflect the announcements you can see on this page - https://datahub-helpcentre.london.cloudapps.digital/updates/announcements/

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to develop?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -118,6 +118,10 @@ const config = {
       'dit:OMISOrder',
     ],
   },
+  helpCentreFeed: {
+    url: `${process.env.HELP_CENTRE_URL}/api/feeds/announcements`,
+    token: process.env.FEED_API_TOKEN,
```
Comment (rafenden) [inline config/index.js]: Is this token specific to the help centre feed or it's shared across different types of content?

File: src/apps/dashboard/views/dashboard.njk [hunk 2]
```diff
@@ -19,18 +19,18 @@
         subHeading: 'Announcements from the Data Hub team. Links open in a new window.',
         feedLimit: 3,
         dataFeed: articleFeed,
-        outboundLinkURL: 'https://uktrade.zendesk.com/hc/en-gb/sections/360000018069-Announcements',
+        outboundLinkURL: 'https://datahub-helpcentre.london.cloudapps.digital/updates/announcements/',
```
Comment (rafenden) [inline src/apps/dashboard/views/dashboard.njk]: I suggest moving this URL to config, same as `helpCentreFeed`, e.g.

```
helpCentreFeed: {
  outboundLinkURL: `${process.env.HELP_CENTRE_URL}/updates/announcements/`,
  ...
}
```

File: src/apps/support/views/feedback.njk [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {% extends "_layouts/template.njk" %}
 
 {% block body_main_content %}
-  Take a look at our <a href="https://uktrade.zendesk.com/hc">help centre</a> for help, guidance and updates.
+  Take a look at our <a href="https://datahub-helpcentre.london.cloudapps.digital/">help centre</a> for help, guidance and updates.
```
Comment (rafenden) [inline src/apps/support/views/feedback.njk]: I think we should use a variable from the config, i.e.
```
Take a look at our <a href="{{ helpCentreUrl }}">help centre</a> for help, guidance and updates.
```

File: config/index.js (hunk 1 above)
Comment (ian-leggett) [reply config/index.js]: Specific to help centre 👍

File: src/apps/dashboard/views/dashboard.njk (hunk 2 above)
Comment (ian-leggett) [reply src/apps/dashboard/views/dashboard.njk]: 👍

File: src/apps/support/views/feedback.njk (hunk 3 above)
Comment (ian-leggett) [reply src/apps/support/views/feedback.njk]: 👍

File: src/apps/dashboard/controllers.js [hunk 4]
```diff
@@ -16,12 +16,15 @@ async function renderDashboard (req, res, next) {
...
+      uri: config.helpCentreFeed.announcementsURL,
+      auth: {
+        'bearer': config.helpCentreFeed.token,
+      },
       json: true,
       timeout: 1000,
     })
-      .then(feed => formatZenArticles(feed))
-      .catch(() => {
+      .then(feed => formatHelpCentreAnnouncements(feed))
```
Comment (yeahfro) [inline src/apps/dashboard/controllers.js]: This isn't being covered by unit tests

File: test/unit/apps/dashboard/transformers.test.js [hunk 5]
```diff
@@ -98,10 +98,10 @@ describe('#formatZenArticles', () => {
 
   context('malformed api responses', () => {
```
Comment (yeahfro) [inline test/unit/apps/dashboard/transformers.test.js]: Do these scenarios actually happen? ie are we writing extra code for something that will never happen?

File: config/index.js (hunk 1 above)
Comment (rafenden) [reply config/index.js]: Then probably we should name it `HELP_CENTRE_ FEED_API_TOKEN`

File: config/index.js [hunk 6]
```diff
@@ -118,6 +118,11 @@ const config = {
       'dit:OMISOrder',
     ],
   },
+  helpCentreFeed: {
+    url: process.env.HELP_CENTRE_URL,
```
Comment (rafenden) [inline config/index.js]: These environment vars are not added to the `README.md` like the rest of variables.

File: src/apps/dashboard/views/dashboard.njk [hunk 7]
```diff
@@ -19,18 +19,18 @@
         subHeading: 'Announcements from the Data Hub team. Links open in a new window.',
         feedLimit: 3,
         dataFeed: articleFeed,
-        outboundLinkURL: 'https://uktrade.zendesk.com/hc/en-gb/sections/360000018069-Announcements',
+        outboundLinkURL: helpHelpCentre.announcementsURL,
```
Comment (rafenden) [inline src/apps/dashboard/views/dashboard.njk]: Typo? `helpHelpCentre` instead of `helpCentre`

File: src/apps/dashboard/controllers.js (hunk 4 above)
Comment (ian-leggett) [reply src/apps/dashboard/controllers.js]: 👍

File: config/index.js (hunk 1 above)
Comment (ian-leggett) [reply config/index.js]: Updated the API token name and in environments inside Vault

**Comment by rafenden** [comment]
Are we no longer using visual tests for homepage as the `Data Hub core pages should visually check data hub home page is correct - chrome.png` file was removed?

**Comment by ian-leggett** [comment]
> Are we no longer using visual tests for homepage as the `Data Hub core pages should visually check data hub home page is correct - chrome.png` file was removed?

Just replaced thanks @rafenden

File: config/index.js (hunk 6 above)
Comment (ian-leggett) [reply config/index.js]: 👍

Comment (ian-leggett) [reply config/index.js]: 👍

File: test/unit/apps/dashboard/transformers.test.js (hunk 5 above)
Comment (ian-leggett) [reply test/unit/apps/dashboard/transformers.test.js]: Probably not, I think Zak had put them into guard if anything changed coming back from the API. Spoke to Pippo about this and he mentioned that we can leave them in for now, happy to remove them to be fair.

**Pull request merged**